vercel.json

```diff
@@ -184,6 +184,11 @@
       "destination": "/reference/rsc/use-server",
       "permanent": true
     },
+    {
+      "source": "/reference/rsc/server-actions",
+      "destination": "https://18.react.dev/reference/rsc/server-actions",
```

Comment:
I don’t think we want people to end up on the 18. site just because they followed a link to server actions docs. Given that the content of the new server functions page is almost the same can we just link to it?